components/director/pkg/graphql/schema.graphql

```diff
@@ -510,4 +522,9 @@ type Mutation {
 
     """If values are not specified, it deletes the label. If values are specified, it deletes values from a specific label."""
     deleteRuntimeLabel(runtimeID: ID!, key: String!, values: [String!]): Label
+
+    # Label Definitions
+    createLabelDefinition(in: LabelDefinitionInput!): LabelDefinition
+    updateLabelDefinition(in: LabelDefinitionInput!): LabelDefinition
+    deleteLabelDefinition(key: String!, force: Boolean=false): LabelDefinition
```

Comment:
Please describe what `force` means. If it means that all labels related to this LabelDefinition are removed, then consider different name like `deleteRelatedLabels` or something like that